client/src/Login.js

```diff
@@ -5,7 +5,7 @@ import './App.css';
 function Login() {
   return (
     <div className="login-container column">  
-    <a href={window.config.baseUrl + '/login/google?target=' + window.config.redirectUrl}><p>log in</p></a>
+      <a href={window.config.baseUrl + '/login/google?target=' + window.config.redirectUrl}><p>log in</p></a>
```

Comment:
`a` is inline element, `p` is block element. Nesting block elements under inline ones causes unexpected behavior sometimes and should be avoided.

Either make the `a` a block element via CSS `display: block` or just drop the `p` and style it properly.